package.nls.json

```diff
@@ -121,5 +121,6 @@
 "mssql.query.ansiWarnings":"Enable SET ANSI_WARNINGS",
 "mssql.query.ansiNulls":"Enable SET ANSI_NULLS",
 "mssql.query.alwaysEncryptedParameterization":"Enable Parameterization for Always Encrypted",
-"mssql.ignorePlatformWarning":"[Optional] Do not show unsupported platform warnings"
+"mssql.ignorePlatformWarning":"[Optional] Do not show unsupported platform warnings",
+"mssql.useNativeCredentials":"Migrate all passwords to native password storage"

```

Comment:
I'd just have this be something like `Use the built-in native credential storage for storing connection passwords`. The "migration" part is just a one-time thing and isn't really what this setting is primarily about.